invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/BooksSearch/components/BookItem/BookItem.js

```diff
@@ -0,0 +1,95 @@
+import React, { Component } from 'react';
+import { Button, Item, Label, Segment, Responsive } from 'semantic-ui-react';
+import { invenioConfig as config } from '../../../../../common/config';
+import { shortenString } from '../../../../../common/helpers';
+
+export default class BookItem extends Component {
+  _render_share_buttons() {
+    return (
+      <div className="book-share-buttons">
+        <Button
+          href="https://www.facebook.com/sharer/sharer.php"
+          circular
+          color="facebook"
+          icon="facebook"
+          size="mini"
+        />
+        <Button
+          href="https://twitter.com/intent/tweet"
+          circular
+          color="twitter"
+          icon="twitter"
+          size="mini"
+        />
+        <Button circular color="linkedin" icon="linkedin" size="mini" />
+      </div>
+    );
+  }
+
+  render() {
+    const cover = 'https://assets.thalia.media/img/46276899-00-00.jpg';
+    const { bookData } = this.props;
```

Comment:
Calling the prop `data` might be enough here too and probably a good idea to also add PropTypes for this component.